torch/fx/passes/runtime_assert.py

```diff
@@ -120,12 +120,13 @@ def add_runtime_asserts(ras):
                     ),
                 )
 
-    for node in graph.nodes:
+    nodes = list(graph.nodes)
+    for i, node in enumerate(nodes[:-1]):
```

Comment:
This makes sure that we only iterate through nodes on the unmodified graph, and do not additionally parse nodes which were just added in this pass.

Reply:
🤔  @Chillee @oulgen is this really the best way to prevent this

Reply:
yeah, this is done at few other places too, see pattern matcher